docs/plugins/main.py

```diff
@@ -7,24 +7,32 @@
 import textwrap
 from pathlib import Path
 from textwrap import indent
+from typing import TYPE_CHECKING
 
 import autoflake
 import pyupgrade._main as pyupgrade_main  # type: ignore
+import requests
 import tomli
 import yaml
 from jinja2 import Template  # type: ignore
 from mkdocs.config import Config
 from mkdocs.structure.files import Files
 from mkdocs.structure.pages import Page
 
-from .conversion_table import conversion_table
-
 logger = logging.getLogger('mkdocs.plugin')
 THIS_DIR = Path(__file__).parent
 DOCS_DIR = THIS_DIR.parent
 PROJECT_ROOT = DOCS_DIR.parent
 
 
+if TYPE_CHECKING:
+    from .conversion_table import conversion_table
+else:
+    from conversion_table import conversion_table
```

Comment:
Not ideal. MkDocs does some `sys.path` magic so the absolute import works but type checkers don't understand it.